VERSION

```diff
@@ -1 +1 @@
-2.33.0+rev1
```

Comment:
Hi. The version file is changed by an automated component we use.
And the meta-balena update is not done like this anyway. You need to update the actual git submodule.
And lastly, 2.37.0 is not production suitable so we'll only update to 2.38 in a few days when that's released